src/Stripe.net/Services/Payouts/StripePayoutService.cs

```diff
@@ -9,6 +9,10 @@ public class StripePayoutService : StripeBasicService<StripePayout>
     {
```

Comment:
it's really awesome that this handles the StripeList<Source> too and lets us kill a class entirely.